### PR TITLE
Initial project structure

### LINKED ISSUE #3: Setup initial project structure

### PR DESCRIPTION
**Pull request by carsongee**
This is just the base folder/file structure of the project.  Need review quickly to continue work.

This will close #3

### CODE REVIEW

**Comment by carsongee** [comment]
I went ahead and added the flask application structure while I was waiting.  Still no functional code, just design that is open to critique

File: lmod_proxy/edx_grades/__init__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+edx_grades = Blueprint(
+    'lmod_proxy_edx_grades',
+    __name__,
+    template_folder='templates/edx_grades',
+    static_folder='static/edx_grades',
+)
+
+
+@edx_grades.route('/', methods=['GET'])
```
Comment (Ferdi) [inline lmod_proxy/edx_grades/__init__.py]: methods=['POST'] ?

Comment (carsongee) [inline lmod_proxy/edx_grades/__init__.py]: yeah, when I write code to handle the post, I'll add that method, planning on using WTForm to validate the edx post and on the get provide a user interface to exercise the form similar to how it is now, but spruced up a bit.

File: lmod_proxy/edx_grades/__init__.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+"""Blue print for handling the ``POST`` from the edx-platform
```
Comment (pwilkins) [inline lmod_proxy/edx_grades/__init__.py]: nit: "Blueprint" is one word

Comment (carsongee) [inline lmod_proxy/edx_grades/__init__.py]: good catch, thanks.

**Pull request merged**